realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -1827,6 +1827,7 @@ public void isValid_removedParent() {
         list.removeAll(SUPPORTED_IS_EMPTY_TYPES);
         list.remove(RealmFieldType.UNSUPPORTED_MIXED);
         list.remove(RealmFieldType.UNSUPPORTED_TABLE);
+        list.remove(RealmFieldType.UNSUPPORTED_DATE);
```

Comment:
But it is also a public API. Having exposed types that are not used at all in the binding is a bit weird, no?